src/routes/notifications/v1/notifications-v2compatible.controller.spec.ts

```diff
@@ -103,9 +106,11 @@ describe('Notifications Controller (Unit)', () => {
   ): Promise<RegisterDeviceDto> => {
     const uuid = faker.string.uuid() as UUID;
     const cloudMessagingToken = faker.string.uuid() as UUID;
-    const timestamp = faker.date.recent();
-    timestamp.setMilliseconds(0);
-    const timestampWithoutMilliseconds = timestamp.getTime();
+    const currentDate = new Date(new Date().getTime());
+    currentDate.setMinutes(
+      currentDate.getMinutes() + REGISTRATION_TIMESTAMP_EXPIRY_MINUTES,
+    );
```

Comment:
This is similar to my [previous comment](https://github.com/safe-global/safe-client-gateway/pull/2192#discussion_r1881988924). Adding the expiration in means it is inherently expired.

Reply:
The code is changed in the new version
https://github.com/safe-global/safe-client-gateway/pull/2192/commits/d0a24a9a6bfca0ad2191fa61eebd8737ac1d6b69